source/installer.py

```diff
@@ -633,7 +659,10 @@ def install(shouldCreateDesktopShortcut: bool = True, shouldRunAtLogon: bool = T
 	# so we shouldn't proceed.
 	# 2. The appropriate executable for nvda.exe will be determined by
 	# which executables exist after copying program files.
-	for f in ("nvda.exe","nvda_noUIAccess.exe","nvda_UIAccess.exe","nvda_service.exe","nvda_slave.exe"):
+	# Some exes are no longer used, but we remove them anyway from legacy copies.
+	# nvda_service.exe was removed in 2017.4 (#7625).
+	# nvda_eoaProxy.exe was removed in 2024.1 (#15544).

```

Comment:
I'm sorry for being painful here, but this, once again, changes the logic by removing EOA proxy way earlier compared to where it used to be removed. Note that it is possible for it to be still running at this point from the UAC screen which requested admin permissions for the installer. This will not only affect Windows 7, but also any more recent version of Windows which was upgraded in place from Win 7, and where NVDA was not reinstalled afterwards, as in that case the more recent support for  ease of access is not applied automatically. Long story short I'd suggest not to remove EOA proxy here, unless these scenarios are tested specifically, which IMHO is not worth the time.

Reply:
Note that Windows 7 is no longer in support, and 2023.3 is the last version with official support. However, I agree we can delay this change to 2024.1 easily